### ISSUE TITLE
Ignore npm-debug.logs 

### ISSUE THREAD

**Issue by paulathevalley** [FormidableLabs/victory-treemap#2]
https://github.com/FormidableLabs/victory-treemap/blob/master/npm-debug.log.13b36952c74bf6403d431569b3afea17

**Comment by ryan-roemer** [comment]
Good catch -- `npm` upgrades have apparently renamed how logs are generated.

Also, side note -- it's a great reminder to _all_ our developers to always **review** the changes going into a commit :wink:
